test/integration/lib/integration-server.ts

```diff
@@ -54,9 +54,7 @@ export class IntegrationServer {
       }
     })
 
-    this.server = this.app.listen(port, () => {
-      console.log(`Integration server listening on port ${port}`)
-    })
+    this.server = this.app.listen(port)
```

Comment:
Is removing thing console.log part of us not wanting to console.log things anymore but wanting to do proper logging in the future?

Reply:
Somewhat I guess - I added it initially when I was debugging service to this test server requests.

Reply:
On that note I don think using pino logger here or anywhere else that is dev only would have any benefit. They aren't run with our other services and aren't going to be consumed by some sort of log aggregator or cloud logging viewer. Just development.

I'm mainly concerned about readability which `pino-pretty` would resolve. Jest has it's own "prettyfied" logs though so if using pino we should not [pipe it](https://github.com/pinojs/pino-pretty?tab=readme-ov-file#usage) and instead add it [in code](https://github.com/pinojs/pino-pretty?tab=readme-ov-file#usage) to preserve that.